docs/authentication.md

```diff
@@ -103,4 +109,7 @@ var sdk = new BoxSDK({
 	}
 });
 var adminClient = sdk.getAppAuthClient('enterprise', 'YOUR-ENTERPRISE-ID');
+// Machine to machine
+var enterpriseApp = sdk.getEnterpriseAppAuthTokens('YOUR-ENTERPRISE-ID', callback);
+var userApp = sdk.getAppUserTokens('APP-USER-ID', callback)
```

Comment:
What is purpose of these functions? I didn't find them used anywhere else and they are not marked as @private so I though that they might be used as alternative to AppAuthClient. 

If they are not supposed to be used by user, shouldn't we marked them as private?